modules/docs/arrow-docs/docs/docs/quickstart/libraries/README.md

```diff
@@ -49,26 +49,26 @@ Datatypes: [`Const`]({{ '/docs/typeclasses/const/' | relative_url }})
 
-### arrow-extras
-
+### arrow-extras 
+// TODO: change readme
 {:.beginner}
 beginner
 
 This library focuses on expanding the helpers provided by typeclasses to existing constructs, like the system collections.
 You can also find more advanced constructs for pure functional programming like the `RWS` datatypes, or transformers.
 
-Dependencies: arrow-typeclasses
-
-Datatypes: [`Cokleisli`]({{ '/docs/datatypes/cokleisli/' | relative_url }}), [`Coreader`]({{ '/docs/datatypes/coreader/' | relative_url }}), [`Ior`]({{ '/docs/arrow/data/ior/' | relative_url }}), [`ListK`]({{ '/docs/arrow/data/listk/' | relative_url }}), [`NonEmptyList`]({{ '/docs/arrow/data/nonemptylist/' | relative_url }}), [`SequenceK`]({{ '/docs/arrow/data/sequencek/' | relative_url }}), [`SortedMapK`]({{ '/docs/arrow/data/sortedmapk/' | relative_url }}), [`StateT`]({{ '/docs/arrow/data/statet/' | relative_url }}), [`WriterT`]({{ '/docs/arrow/data/writert/' | relative_url }}), [`Coproduct`]({{ '/docs/arrow/data/coproduct/' | relative_url }}), [`EitherT`]({{ '/docs/arrow/data/eithert/' | relative_url }}), [`Kleisli`]({{ '/docs/arrow/data/kleisli/' | relative_url }}), [`MapK`]({{ '/docs/arrow/data/mapk/' | relative_url }}), [`OptionT`]({{ '/docs/arrow/data/optiont/' | relative_url }}), [`Reader`]({{ '/docs/arrow/data/reader/' | relative_url }}), [`SetK`]({{ '/docs/arrow/data/setk/' | relative_url }}), [`State`]({{ '/docs/arrow/data/state/' | relative_url }}), [`Validated`]({{ '/docs/arrow/data/validated/' | relative_url }})
+Dependencies: arrow-core-data
+// TODO: Change locations in Readme before merge
```

Comment:
FIX before merge!

Reply:
The relative links will be changed anyways so I will create the issue push another commit without the Todo's and we're good